### DISCUSSION TITLE
No such file or directory

### DISCUSSION THREAD

**Discussion by Vm900** [company-mode/company-mode#1195] [Q&A] [answered]
Hi,
I've been trying to install  company manually.I copied the entire folder to my .emacs.d/lisp
and added (add-to-list 'load-path "~/.emacs.d/lisp/") ,(load 'company-mode) to my .emacs file.But i keep getting the error: file missing :Cannot open load file,No such file or directory
Help.Thanks

**Comment by cyrialize** [comment, marked as answer]
Here's a good write up of how load, load-file, autoload, and require works: http://ergoemacs.org/emacs/elisp_library_system.html

Basically, with load it first tries to find a file by going through the load-path and looking for a file that has that file name and ends with elc, then el and then just the file name. 

So, it's looking for:
- company-mode.elc
- company-mode.el
- company-mode

The reason why this doesn't work is because the base company-mode file is just company. So, to fix this you can do (load 'company). It's better to use require though, so I'd recommend (require 'company)
